shell/browser/ui/devtools_ui.cc

```diff
@@ -118,6 +140,162 @@ class BundledDataSource : public content::URLDataSource {
   }
 };
 
+class ThemeDataSource : public content::URLDataSource {
```

Comment:
I'd prefer if we followed the file conventions in Chromium here and made this a separate file like [devtools_ui_data_source.cc](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/webui/devtools/devtools_ui_data_source.cc)

Reply:
Do you mean both `BundledDataSource `and `ThemeDataSource ` need to be separated into separate files?

Reply:
I had some initial confusion reading through this so I think separate files like `devtools_ui_bundled_source` and `devtools_ui_theme_source` could be helpful. Especially given that [theme_source.h](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/webui/theme_source.h;drc=36147a880112f3dfd7a288309c83a483b520dd7c) is its own file upstream.

Reply:
@cucbin yes - I agree with @samuelmaddock here.

Reply:
I have considered this before, but considering that there are duplicate code of anonymous methods, so final put them in one file.
But I also agree this, had changed it.